src/nvim/os/shell.c

```diff
@@ -390,7 +390,7 @@ static bool out_data_decide_throttle(size_t size)
   static uint64_t   started     = 0;  // Start time of the current throttle.
   static size_t     received    = 0;  // Bytes observed since last throttle.
   static size_t     visit       = 0;  // "Pulse" count of the current throttle.
-  static char       pulse_msg[] = { ' ', ' ', ' ', '\0' };
+  static char       pulse_msg[] = { '.', ' ', ' ', '\0' };
```

Comment:
As mentioned before, this makes the pulse less animated. It looks more like a blink.

Reply:
Any idea about not ending up with an empty line then?

Reply:
what's wrong with an empty line? An unchanging dot isn't going to be any more intuitive than an empty line.

Reply:
ok, reverted.